### DISCUSSION TITLE
Update firebase database sdk to 20.3.0

### DISCUSSION THREAD

**Discussion by CarlosCMD** [Sketchware-Pro/Sketchware-Pro#1049] [Q&A]
The current outdated database sdk does not include new features like the get() method introduced in [19.6.0]. I've tried downloading the latest one but the default lib overwrites the new one.

**Comment by j-px** [comment]
While built-in library upgrades are annoying, the fix to the problem you mentioned at the end is easy: Use what's described in #696.
